.github/workflows/ci.yml

```diff
@@ -182,4 +182,5 @@ jobs:
       - name: Deploy
         run: yarn deploy --verbose
         env:
+          CI: true
```

Comment:
Setting the environment variable `CI` to `true` has the same effect as writing `enabled: true` in the deploy configuration file.

https://github.com/dfreeman/ember-cli-deploy-git-ci#configuration